### ISSUE TITLE
Having Trouble Running Simple Filter Command

### ISSUE THREAD

**Issue by Jalapenobadger** [wdecoster/nanofilt#44]
Hey, I think I installed this correctly, because putting in NanoFilt -h brings up the Help, but when I try to run : 

gunzip -c fastqtest.fastq.gz | NanoFilt -q 10 | gzip > highQuality-reads.fastq.gz

I get:

 'gunzip' is not recognized as an internal or external command,

I've cd'd to the directory with this file in it, and I'm using miniconda3. Ultimately, I am just trying to filter out fastq data smaller than 5000bp, I'm just running this command to test it.

I apologize for whatever very obvious thing I'm missing, I am new to bioinformatics.
Thanks

**Comment by wdecoster** [comment]
Gunzip is not something I developed, and should be already installed on your computer. It's the inverse of gzip and is used to decompress gzipped files. Which operating system are you using? Does 'zcat yourfile.fastq.gz | NanoFilt (etc) work?

**Comment by Jalapenobadger** [comment]
Ah, I am using Windows 10, I just assumed it was a command part of NanoFilt. So I guess I need to use a unzip software which will work in the command line. Thanks for responding so quick!

**Comment by Jalapenobadger** [comment]
Is there a preferred unzip tool for Windows that would work with Anaconda and NanoFilt? Sorry for all the questions.

**Comment by wdecoster** [comment]
Don't worry about it, I wasn't aware that gunzip doesn't exist on Windows 10. Is it WSL you are using?

**Comment by Jalapenobadger** [comment]
I have simply been trying to get it to work with miniconda on a straight up
Windows machine, but I'm looking into WSL now as that might be easier in
the long term, yeah.

**Comment by Jalapenobadger** [comment]
I believe that I've got it working. Thank you for your help! I'm not 100% sure it is actually doing what I hope it is, but I have succeeded in getting it running on Anaconda through WSL in a Jupyter notebook, with the generic script:

 gunzip -c reads.fastq.gz | NanoFilt -q 10 | gzip > highQuality-reads.fastq.gz

The output is smaller than the input and is also a properly opening fastq file so I'm sure when I get to check it this will work.
